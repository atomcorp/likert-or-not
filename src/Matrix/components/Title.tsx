import * as React from 'react';
// import css from '../Matrix.css';

import {TitleProps} from '../definitions';

const Title = (props: TitleProps) => <h2>{props.children}</h2>;

export default Title;
