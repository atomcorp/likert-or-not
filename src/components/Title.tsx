import * as React from "react";
// import css from '../index.css';

import { TitleProps } from "../typings";

const Title = (props: TitleProps) => <caption>{props.children}</caption>;

export default Title;
