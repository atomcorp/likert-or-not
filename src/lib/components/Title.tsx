import * as React from 'react';
// import css from '../Matrix.css';

import {TitleProps} from '../definitions';

const Title = (props: TitleProps) => <caption>{props.children}</caption>;

export default Title;
