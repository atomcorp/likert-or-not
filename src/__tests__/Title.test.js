import React from 'react';
import Title from '../components/Title';
import renderer from 'react-test-renderer';

test('Title prints text in caption', () => {
  const component = renderer.create(<Title>The title</Title>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
