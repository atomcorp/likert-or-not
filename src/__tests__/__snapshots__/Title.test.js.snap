// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title prints text in caption 1`] = `
<caption>
  The title
</caption>
`;
